tests_e2e/orchestrator/runbook.yml

```diff
@@ -54,6 +54,9 @@ variable:
       - publish_hostname
       - recover_network_interface
       - cgroup_v2_disabled
+  # agent_wait_for_cloud_init and initial_agent_update runs on custom image, as we don't update the custom image daily in the test run, So we are skipping but make sure to run when we added breaking changes like new lisa version, pipeline changes etc
```

Comment:
Your concern about people forgetting these tests is valid. However, this comment won't help with that. 

I'd suggest defining a new test suite ("manual_tests"?) and adding these tests to that suite. Then updating our documentation asking to ensure running those tests after updating LISA and at least once before any release.

Could be done on a separate PR, if you prefer so.